### PR TITLE
Add title to contributor details page

### LINKED ISSUE #37: Add a `<title>` tag to the pages with the person's name e.g.
### What feature do you want to see added?

Without, it looks pretty weird:

Screenshot 2023-11-30 at 18 25 22

### Upstream changes

_No response_

### PR DESCRIPTION
**Pull request by sridamul**
Fixes #37 

### Initial Preview
initial contributor page title

### After Changes
contributor page title

### CODE REVIEW

**Comment by krisstern** [comment]
Hi @sridamul Please follow the formula for pull request titles: it should always start with a verb.

**Comment by sridamul** [comment]
@krisstern Okay! Thanks for the correction.

File: src/templates/contributor-details.jsx [hunk 1]
```diff
@@ -125,4 +131,4 @@ export const pageQuery = graphql`
       }
     }
   }
-`
+`
```
Comment (krisstern) [inline src/templates/contributor-details.jsx:134]:
```suggestion
`

```

Comment (krisstern) [reply src/templates/contributor-details.jsx:134]: Please make sure there is always a blank line at the EOF.

**Review by krisstern — CHANGES_REQUESTED** [review]
@sridamul Could you please clean up the indentation to make it consistent as it is not looking nice now?

**Comment by sridamul** [comment]
@krisstern Sorry for the confusion! The blank line at the end of the file is automatically deleted due to auto-formatting in my IDE.

**Comment by krisstern** [comment]
Actually I do not favor an approach with `react-helmet`, so I will close this issue as unplanned for now. @sridamul Next time before working on an issue do ask if someone else has already started work on it or if there is a preferred approach.

**Comment by sridamul** [comment]
Hi @krisstern, I was under the impression that there were no assigned individuals actively working on the issue. That's why I took the initiative to address it. In the future, I'll make sure to inquire about the preferred approach before proceeding.

**Comment by krisstern** [comment]
No worries, this is a new repo we have just started so maybe we would still like to add features as they come. But sometimes when a project starts we would like to take more ownership of it.

**Pull request closed without merging**